src/csharp/Microsoft.Spark.E2ETest/SparkFixture.cs

```diff
@@ -120,6 +121,9 @@ private void BuildSparkCmd(out string filename, out string args)
             string jarDir = $"{scalaDir}{sep}{jarPrefix}{sep}target";
             string assemblyVersion = Assembly.GetExecutingAssembly().GetName().Version.ToString(3);
             string jar = $"{jarDir}{sep}{jarPrefix}-{assemblyVersion}.jar";
+            string packagesArg = $@"--packages {string.Join(',', new List<string>() {
+                "io.delta:delta-core_2.11:0.3.0"
```

Comment:
This list is only one element ?  Should we just do "--packages io.delta:delta-core_2.11:0.3.0" for now ?

Reply:
Ok